<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Форма входа</string>
  </property>
  <widget class="QFrame" name="registerframe">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 244, 244);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLineEdit" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Электронная почта</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backregister">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLineEdit" name="NameLabel2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Имя пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordLabel2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegistrationButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>registerframe</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
